CONTRIBUTING.md

```diff
@@ -95,24 +95,18 @@ JEKYLL_ENV=local make build  # Do a clean build of the site to the _site directo
 diff -ruN _site _site.bak  # Compare the generated sites (-r for recursive, -u for unified, -N for new file)
 ```
 
-## Compatibility Matrix Data
+## Bitcoin Feature Matrix Data
```

Comment:
@TorokNRoll do you want to include the new process for user-submitted contributions for the matrix here?